lemminx-liberty/src/it/variable-processing-ol-it/pom.xml

```diff
@@ -84,7 +84,7 @@
                     <runtimeArtifact>
                         <groupId>io.openliberty</groupId>
                         <artifactId>openliberty-runtime</artifactId>
-                        <version>24.0.0.11</version>
+                        <version>25.0.0.2</version>
```

Comment:
For future reference, we do not need to update this version when updating cached data. It's important to prove we can generate for older runtimes and everything still works.